test/types/client-options.test-d.ts

```diff
@@ -616,3 +616,20 @@ expectError<errors.ConfigurationError>(
     })
   )
 }
+
+/**
+ * `context` option
+ */
+expectType<Client>(
+  new Client({
+    node: 'http://localhost:9200',
+    context: { hello: 'world' }
+  })
+)
+
+expectError<errors.ConfigurationError>(
+  new Client({
+    node: 'http://localhost:9200',
+    context: 'hello world'
```

Comment:
could you add a test for `context` reading within a `response` callback?

Reply:
It's already present in `test/types/client.test-d.ts` :)